corehq/apps/accounting/subscription_changes.py

```diff
@@ -91,13 +92,19 @@ def privilege_to_response_function(cls):
             privileges.COMMCARE_LOGO_UPLOADER: cls.response_commcare_logo_uploader,
             privileges.ADVANCED_DOMAIN_SECURITY: cls.response_domain_security,
             privileges.PRACTICE_MOBILE_WORKERS: cls.response_practice_mobile_workers,
+            privileges.MOBILE_WORKER_CREATION: cls.response_mobile_worker_creation,
```

Comment:
Does this function get executed locally?  I'm guessing it won't because ```privileges.MOBILE_WORKER_CREATION``` isn't used in any software plans, but I could be wrong

Reply:
Ah yep. I meant to take that out. It's basically a useless line since I added the get_response override